server/src/main/java/org/apache/iotdb/db/query/reader/chunkRelated/DiskChunkReader.java

```diff
@@ -70,8 +72,18 @@ public TimeValuePair current() {
     return TimeValuePairUtils.getCurrentTimeValuePair(data);
   }
 
+  @Override
+  public boolean hasNextBatch() throws IOException {
+    return false;
+  }
+
+  @Override
+  public BatchData nextBatch() throws IOException {
+    return null;
+  }
+
```

Comment:
The fake implementation is for future use? Why should the DiskChunkReader implement the IBatchReader?

Reply:
We could remove this fake implementation currently. I plan to remove TimeValuePair in anywhere in the future.